README.md

```diff
@@ -1 +1,27 @@
-# jv-homework-template
+# jv-strategy
+
+---
+Imagine you have сhain of stores. You need to implement discount system for people 
+in some special events of a year like New Year.
+
+In this task we'll need help of Strategy pattern.
+First of all, you need to create `DiscountStrategy` interface 
+with method `calculatePriceWithDiscount()` with one parameter `originalPrice` and 
+return type `double`. It must be located in `core.basesyntax.strategy` package.
+
+You must create its implementations (with the same names) with such discounts:
+
+| Implementation name |Discount |
+| :---: | :---: |
+| NewYearDiscountStrategy| 20 |
+| BirthdayDiscountStrategy| 33 |
+| BlackFridayDiscountStrategy| 45 |
```

Comment:
You should have one class called `strategy`. And you should have one interface and multiple implementations, but they are not strategies. For example, you can have `DiscountService` or `DiscountCalculator` and have several implementations. But you still should have one class DiscountStrategy